frontend/src/components/jobs/result/utils/JobOverview.jsx

```diff
@@ -15,14 +15,14 @@ export default function JobOverview({ isRunningJob, job, refetch }) {
         <strong>Analyzers Report</strong>
         <Badge className="ms-2">
           {job.analyzers_to_execute?.length} /&nbsp;
-          {job.analyzers_requested?.length || "all"}
+          {job.analyzers_requested?.length}
         </Badge>
       </div>,
       <div className="d-flex-center">
         <strong>Connectors Report</strong>
         <Badge className="ms-2">
           {job.connectors_to_execute?.length} /&nbsp;
-          {job.connectors_requested?.length || "all"}
```

Comment:
the result of this would be:
* now you have `0` instead of `all` when running playbooks. So this is still wrong
* and more now if you run all the analyzers, you would have `0` instead of `all`.
So the overall result of this change is that is worse than before.

I think that you should get here when you are running playbooks. In that case, manage this differently: the `connectors_requested` and `analyzers_requested` would not exist in that case. I think you could simply use `connectors_to_execute` and `analyzers_to_execute` instead.

Reply:
Works? 
![image](https://user-images.githubusercontent.com/60684641/189109389-e9886384-9920-4e0a-bef9-c903b89561f0.png)

Reply:
let me know if you want anything different when no connectors are added to the playbook.

Reply:
Would also take care of this portion the same way (plugins executed)
![image](https://user-images.githubusercontent.com/60684641/189111527-05b959b2-847b-45d2-8aa5-1db9ca844315.png)

Reply:
> Works?
that's good

Reply:
Seems alright to me now. 
![image](https://user-images.githubusercontent.com/60684641/189174217-8cfa628b-bfb4-4142-a46f-060629fa890c.png)